scipy/stats/_continuous_distns.py

```diff
@@ -1520,6 +1520,10 @@ def _ppf(self, q, a):
         fac = sc.gammainccinv(a, 1-abs(2*q-1))
         return np.where(q > 0.5, fac, -fac)
 
+    def _entropy(self, a):
+        h = np.log(2) + sc.loggamma(a) + a + (1 - a) * sc.digamma(a)
```

Comment:
```suggestion
        h = np.log(2) + sc.gammaln(a) + a + (1 - a) * sc.digamma(a)
```

`loggamma` is in my understanding mostly useful for complex numbers even if the result is the same here.

Reply:
I don't think it really matters. From what I understand, using loggamma doesn't affect the speed or accuracy, so we might as well leave it.